src/main/solution_space/path_finder_test_system.py

```diff
@@ -130,11 +132,12 @@ def get_result_table(self, title: str) -> Optional[PrettyTable]:
         table = PrettyTable(field_names=[e.value for e in TEST_INPUT] +
```

Comment:
Instead of adding new value to TEST_INPUT, you can just do 
field_names = ["index"] + [e.value for e in TEST_INPUT]